apps/nextra/components/graphql/NonDynamicGraphQL.tsx

```diff
@@ -1052,6 +1052,20 @@ interface FormValues {
 export interface GraphQLEditorProps {
   network: Network;
   query: string;
+  /**
+   * A set of variable replacements to use in the editor. Replaces $variables with the same name.
+   * Ex.
+   * {
+   *   "owner_address": "0xaa921481e07b82a26dbd5d3bc472b9ad82d3e5bfd248bacac160eac51687c2ff",
+   *   "offset": 0
+   * }
+   */
+  variables?: string;
+  /**
+   * Useful for having multiple GraphQLEditors on a single page only display one view each.
+   * Otherwise having multiple editors on the same page will open a tab for each of them.
+   */
+  disableTabs?: boolean;
```

Comment:
I think this removed more than just the tabs... There's also no network selector, no docs explorer, or no history explorer anymore (https://preview.aptos.dev/en/docs/components/graphql-editor)

Reply:
They're rendered very strangely <img width="1149" alt="Screenshot 2024-05-04 at 10 03 34 AM" src="https://github.com/aptos-labs/developer-docs/assets/31301117/313dde42-1d24-46e5-a2af-7ba05dde5ea8">

Reply:
From some testing, here's what I've noticed:
1. Locally, my editors always extend to the full width of the page. (Even when opened in a fresh incognito window)
2. *When tabs are disabled* the first editor page opened is missing the "Mainnet" selector bar, which is a problem.
<img width="1413" alt="image" src="https://github.com/aptos-labs/developer-docs/assets/163934542/528ac03c-e55d-4400-978e-d2951f5d7c23">
3. If I enable tabs though, currently switching to another editor keeps all pre-existing tabs. This creates a very cluttered experience where it's unclear why all these tabs are here, and they must be manually closed. 
<img width="1413" alt="image" src="https://github.com/aptos-labs/developer-docs/assets/163934542/89a6dd84-e94b-4575-9738-bae85f838f9b">

I'm currently looking into how to remove the tab history, which I think would be a better solution overall.

In the meantime - @hariria I set the `Coin Balances` page to HAVE tabs. Can you share a screenshot of the way that looks in Vercel? (I'm not currently able to replicate the editor only taking up half the screen as in your screenshot above so I'm not sure if it's caused by `disableTabs`).

Reply:
Hmm they look the same on all pages for me. Can you try removing the disableTabs prop?
![Uploading Screenshot 2024-05-09 at 12.40.05 PM.png…]()

Reply:
@jmintuitive what happens when you don't include the variables and just have the query? Do the tabs still appear? Seems odd that all of those tabs are showing up

Reply:
Just using the query still causes the tabs to appear when swapping between pages with the editor on it. 
<img width="1157" alt="image" src="https://github.com/aptos-labs/developer-docs/assets/163934542/1441ef46-a64d-41e8-aa74-f38c3770ad6d">

When I was originally experimenting with having all editors on the same page - every editor had every tab on page-load, just with a different focused tab.

That's why I think the issue is that there's a shared context being used for tabs which doesn't make sense for our use case. These are meant to be standalone examples, not things that get remembered long-term. So I want to figure out how to make each component have it's own context, or forget previous tabs. (Note: Refreshing the same page over and over doesn't spawn new tabs - but swapping back and forth between two does, even though the contents of the tabs are the same each time).

Reply:
(And for completeness, entirely removing the "disableTabs" produces the following outputs:
1. Incognito first editor you open: (Ideal UI in my opinion if we can get it to consistently show up :) )
<img width="1157" alt="image" src="https://github.com/aptos-labs/developer-docs/assets/163934542/00ff4d9d-c405-4905-a8ab-155e2267c76d">


2. After swapping from another page with an editor: (Tabs accumulating)
<img width="1157" alt="image" src="https://github.com/aptos-labs/developer-docs/assets/163934542/55462827-46b1-4094-8ddd-797c9d7729c2">

Reply:
Fixed the issue :) - Turns out the tabs and other header content were being formatted with a `flex`, so when tabs were removed with a `null` the header appeared on the left in the 1st position. Instead of that, if we add an empty div, the formatting looks like we expect it to :)

<img width="1147" alt="image" src="https://github.com/aptos-labs/developer-docs/assets/163934542/f3c312bc-69b4-4306-8099-a4271b53faaa">